src/js/tech/html5.js

```diff
@@ -217,22 +237,36 @@ class Html5 extends Tech {
           !elTracks.addEventListener) {
         return;
       }
-      const listeners = {
-        change(e) {
+
+      const listeners = {};
+
+      listeners.change = (e) => {
+        if (!this[`forceNative${props.capitalName}Override_`]) {
           techTracks.trigger({
             type: 'change',
             target: techTracks,
             currentTarget: techTracks,
             srcElement: techTracks
           });
-        },
-        addtrack(e) {
+        }
+      };
+
+      listeners.addtrack = (e) => {
+        if (this[`forceNative${props.capitalName}Override_`] && elTracks.addTrack) {
```

Comment:
why would `elTracks` not have an `addTrack` method?

Reply:
I'm not sure, it seemed safer to check to see if a method existed on something before using it.